### PR TITLE
Basic .nvmrc file support.

### PR DESCRIPTION
**Pull request by wsobolewski**
#593 fix

### CODE REVIEW

File: src/nvm.go [hunk 1]
```diff
@@ -160,10 +160,14 @@ func install(version string, cpuarch string) {
   }
 
   if version == "" {
+		if file.Exists(".nvmrc") {
+      version = nvmrc()
+		} else {
```
Comment (duszekmestre) [inline src/nvm.go:165]: Pls align indentation with other code

File: src/nvm.go [hunk 2]
```diff
@@ -160,10 +160,14 @@ func install(version string, cpuarch string) {
...
     fmt.Println("\nInvalid version.")
     fmt.Println(" ")
     help()
     return
+		}
```
Comment (duszekmestre) [inline src/nvm.go:170]: Pls align indentation with other code

File: src/nvm.go [hunk 3]
```diff
@@ -451,6 +461,15 @@ func useArchitecture(a string) {
   }
 }
 
+func nvmrc() string {
+  line, err := file.ReadLines(".nvmrc")					
+  if err == nil {
```
Comment (duszekmestre) [inline src/nvm.go:466]: What if error occured? Maybe you should print error message in that case?

**Review by duszekmestre — CHANGES_REQUESTED** [review]

**Comment by aakash-verma-aurea** [comment]
@duszekmestre any more help needed? Or are we good to merge this?

**Comment by celioreyes** [comment]
Was thinking of implementing this since I noticed `.nvmrc` support wasn't already implemented.

Anything in particular blocking this PR from being merged?

**Comment by it-rec** [comment]
We have 2022 and this is still not merged nor any other comment after the fixes was made. It would be superb to get this soon!

**Comment by coreybutler** [comment]
While I appreciate the contribution, I do not plan to merge this. Please see the [reasons why](https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#why-isnt-nvmrc-supported-why-arent-some-nvm-for-macoslinux-features-supported).

**Pull request closed without merging**